requirements/base.txt

```diff
@@ -38,6 +38,8 @@ oauthlib==3.2.2
     # via
     #   kubernetes
     #   requests-oauthlib
+packaging==24.1
+    # via -r requirements/base.in
```

Comment:
Consistency nit: the other `requirements/*.txt` use `packaging==23.2`

Reply:
And don't we need it in `tutor@master` too?

Reply:
Thanks @pomegranited for the review.

Synced the requirements from base.txt to the other *.txt files. It's consistent now.

> And don't we need it in tutor@master too?

The commit which used this package is only present in `nightly` right now. I am unsure about the process here, but I am assuming that changes in `nigtly` will be syced to `master` at some point ?

Reply:
I think it was a mistake to merge that PR in nightly only. Can you change your PR to target master please?

Reply:
@regisb PR rebased to target master.